.firebase.json

```diff
@@ -1,7 +1,12 @@
 {
   "emulators": {
     "auth": {
+      "host": "0.0.0.0",
```

Comment:
no, thats not working and the firebase auth emulator is not accessible from my browser.
but it accessible from container env --
![Screenshot 2023-07-13 at 3 07 00 PM](https://github.com/oppia/oppia/assets/91013793/c7cabe2e-80e8-478e-a259-b2b2120893d9)

which means not accessible from my host network, but functioning in docker env. so that's why we need to bind it with 0.0.0.0